### DISCUSSION TITLE
Chess 4.6 possible?

### DISCUSSION THREAD

**Discussion by betzburger** [kej715/DtCyber#108] [General]
First of all, I am sorry but I do not have any experience with this type of Computer. I never saw a running version of Chess 4.6 (Atkin/Slate) anywhere and I wonder if it may be possible to run it on DtCyber. Atkin made the source code available to the public and it is written in COMPASS. I have the source code but I have no idea if we can do something with it here. It would be sad if such a classic program is lost forever. Any comments?

**Comment by kej715** [comment]
Hi Peter,

We have a version of CHESS 4.7 buried in our archives, and I've not seen
that running either. It should be possible to get the 4.6 version you have
running, and it would be very worthwhile to invest the time in trying. Can
you provide a link to the source, or try emailing a copy to my mailbox at
***@***.***?

thanks!
Kevin

**Reply by betzburger** [reply]
Hi Kevin, okay, I can't see your mail in your post but I will send the source code to the admin of your nostalgiccomputing webpage.
If there is anything I can do, I am happy to help.

Peter

**Comment by KC5VDJ** [comment]
When drop is issued to it when running on the console, it isn't restoring dsd.  Other issues too, but it does run.  Duchess on the mvs tk5 dist has issues too, so...

**Comment by kej715** [comment]
The command for exiting CHESS on the console is KILL., not DROP.

**Reply by KC5VDJ** [reply]
Thanks.

**Reply by betzburger** [reply]
KC5VDJ... Look at that. Those hams are everywhere ;-) My callsign is DD2ZG.

**Comment by KC5VDJ** [comment]
.... .. .... ..
